src/main/java/CardGenerator.java

```diff
@@ -0,0 +1,46 @@
+import model.Card;
+
+import java.util.Random;
+
+public class CardGenerator {
```

Comment:
Can you use the default access modifier?

Reply:
Fixed